MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -215,15 +215,15 @@ class TestCarPlayManagerDelegate: CarPlayManagerDelegate {
     public var trailingBarButtons: [CPBarButton]?
     public var mapButtons: [CPMapButton]?
 
-    func carPlayManager(_ carPlayManager: CarPlayManager, leadingNavigationBarButtonsCompatibleWith traitCollection: UITraitCollection, in template: CPTemplate) -> [CPBarButton]? {
+    func carPlayManager(_ carPlayManager: CarPlayManager, leadingNavigationBarButtonsCompatibleWith traitCollection: UITraitCollection, in template: CPTemplate, for activity: CarPlayActivity) -> [CPBarButton]? {
```

Comment:
Thanks, not sure how I missed this earlier.

Reply:
it's the optionality. It results in no help from the compiler and the other test failures have been reducing our reliance on the tests :( trying hard to fix that up